docs/reference/admin-app-v4/system-administrators/readme.md

```diff
@@ -0,0 +1,1162 @@
+4. [Configuration](#configuration)
+   - [Backend API Configuration](#backend-api-configuration)
+   - [Frontend Configuration](#frontend-configuration)
+   - [Database Configuration](#database-configuration)
+   - [Authentication Configuration](#authentication-configuration)
+5. [Security Considerations](#security-considerations)
+6. [Troubleshooting](#troubleshooting)
+7. [Maintenance](#maintenance)
+
+## Overview
+
+The Ed-Fi Admin App is a user interface for managing Ed-Fi Technology Suite deployments. It consists of:
+
+- **Frontend**: React-based single-page application (SPA)
+- **Backend API**: Node.js/NestJS application
+- **Database**: PostgreSQL database for application data
+- **Authentication**: OpenID Connect (OIDC) integration (typically Keycloak)
+
+### Required Components
+
+- **PostgreSQL Database** (Required)
+- **OIDC Provider** (Required) - Keycloak or similar
+- **Reverse Proxy** (Recommended) - NGiNX, IIS, or similar
+
+### Optional Components
+
+- **Yopass** (Optional) - For sharing encrypted secrets
+- **Memcached** (Optional) - Required only if using Yopass
+
+## Prerequisites
+
+### System Requirements
+
+- **Operating System**: Windows Server 2019+, Linux (Ubuntu 20.04+, RHEL 8+), or similar
+- **Memory**: Minimum 4GB RAM, Recommended 8GB+
+- **Storage**: Minimum 10GB free space
+- **Network**: HTTPS capability (SSL/TLS certificates)
+
+### Software Dependencies
+
+- **Node.js**: Version 18.0.0 or higher
+- **PostgreSQL**: Version 12 or higher
+- **SSL/TLS Certificates**: For HTTPS endpoints
+
+## Installation Methods
+
+## Docker Installation (Recommended)
+
+Docker provides the easiest deployment method with consistent environments across platforms.
+
+### Docker Prerequisites
+
+- Docker Engine 20.10+ or Docker Desktop
+- Docker Compose v2.0+
+- Git (for cloning the repository)
+
+### Quick Start
+
+1. **Clone the repository**:
+
+   ```bash
+   git clone https://github.com/Ed-Fi-Alliance-OSS/AdminApp-v4.git
+   cd AdminApp-v4/compose
+   ```
+
+2. **Create environment configuration**:
+
+   ```bash
+   cp .env.example .env
+   # Edit .env file with your specific settings
+   ```
+
+3. **Generate SSL certificates**:
+
+   ```bash
+   # On Linux/WSL/Git Bash:
+   cd ssl && ./generate-certificate.sh && cd ..
+   ```
+
+4. **Start core services**:
+
+   ```bash
+   # PowerShell (Windows):
+   .\up.ps1
+   
+   # Linux/macOS:
+   mkdir -p logs
+   docker network create edfiadminapp-network --driver bridge
+   docker-compose up -d
+   ```
+
+5. **Start Admin App services**:
+
+   ```bash
+   # PowerShell (Windows):
+   .\up.ps1 -AdminApp
+   
+   # Linux/macOS:
+   cd adminapp
+   docker-compose up -d
+   ```
+
+### Production Docker Deployment
+
+For production environments, modify the following:
+
+1. **Environment Variables** (`.env`):
+
+   ```bash
+   # Use strong, unique passwords
+   POSTGRES_PASSWORD=your-secure-password
+   KEYCLOAK_ADMIN_PASSWORD=your-keycloak-password
```

Comment:
My PR in review uses two new env vars for Keycloak: https://github.com/Ed-Fi-Closed/Ed-Fi-Admin-App/pull/91/files#diff-425f5e97be654c8dff796d7618a39664b59fdc48768290bf6dd5a7d5d0215083